pkg/jobrunaggregator/jobrunaggregatorapi/types_row_jobrun.go

```diff
@@ -0,0 +1,65 @@
+package jobrunaggregatorapi
+
+import "time"
+
+const (
+	LegacyJobRunTableName     = "JobRuns"
```

Comment:
Am I following that LegacyJobRunTableName contains all job runs, and DisruptionJobRunTableName contains only those with disruption data?

If so, given this is a permanent table we are not trying to phase out, it doesn't feel like it's Legacy, rather AllJobRunsTableName or equivalent.